pyproject.toml

```diff
@@ -80,7 +80,7 @@ all_extras = [
   "arch<6.3.0,>=5.6",
   "cloudpickle",
   "dash!=2.9.0",
-  "dask",
+  "dask<2023.12.2",
```

Comment:
Note: it'll trigger all the validate extras jobs, but it doesn't cover `all_extras`. Only component specific dependency sets and `all_extras_pandas2` are covered.

https://github.com/sktime/sktime/blob/5ab219917e23233bef2ee17bce600f5996edaeec/.github/workflows/validate_extras.yml#L23-L33

Reply:
I hoped a change to `pyproject.toml` would trigger everything